### DISCUSSION TITLE
remark-directive error - parsing nested lists

### DISCUSSION THREAD

**Discussion by mikeswain** [remarkjs/remark#663] [General] [answered]
Hi Guys,
Great package by the way.
I'm having trouble using container directives - I think I've got it down to a minimal example:
```
:::container
1. a
    1. b
# c
:::
```
Uncaught TypeError: Cannot set property 'next' of undefined
    at subcontent (subtokenize.js:179)
    at subtokenize (subtokenize.js:60)
    at postprocess (postprocess.js:6)
    at fromMarkdown (index.js:24)
    at parse (index.js:13)
    at Function.parse (index.js:267)
    at ReactMarkdown (react-markdown.js:42)

Anyone seen anything similar? not 1. b is indented 4 spaces to continue numbering (if you indent 2 its fine)
code looks like its trying to set an undefined member (token is null/undefined) I think there's a mistake in the link list code

```
// Help GC.
  tokenizer.events = token._tokenizer = token.next = undefined // Do head:


```

**Comment by ChristianMurphy** [comment, marked as answer]
It's a known issue, there's ongoing discussion at https://github.com/micromark/micromark-extension-directive/issues/3
Insights and PRs are welcome!
